crates/proof-of-sql/src/sql/proof/query_proof.rs

```diff
@@ -54,8 +54,8 @@ impl<CP: CommitmentEvaluationProof> QueryProof<CP> {
 
         let alloc = Bump::new();
         let mut result_builder = ResultBuilder::new(table_length);
-        expr.result_evaluate(&mut result_builder, &alloc, accessor);
-        let provable_result = result_builder.make_provable_query_result();
+        let result_cols = expr.result_evaluate(&mut result_builder, &alloc, accessor);
+        let provable_result = result_builder.make_provable_query_result(&result_cols);
```

Comment:
Since `make_provable_query_result` is just a wrapper around `ProvableQueryResult::new`, perhaps it's tidier to just call `ProvableQueryResult::new` here, which simplifies the `ResultBuilder`. (And essentially removes the "Result" portion of it.)